NetKAN/VesselView-UI.frozen

```diff
@@ -1 +0,0 @@
-VesselView-UI
```

Comment:
Please restore this file. - It's the placeholder for the virtual mod that you're providing